payments/invoice/direct.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Swedbank Pay Payments Invoice Direct
+sidebar:
+  navigation:
+  - title: Invoice Payments
+    items:
+    - url: /payments/invoice
+      title: Introduction
+    - url: /payments/invoice/direct
+      title: Direct
+    - url: /payments/invoice/redirect
+      title: Redirect
+    - url: /payments/invoice/seamless-view
+      title: Seamless View
+    - url: /payments/invoice/after-payment
+      title: After Payment
+    - url: /payments/invoice/other-features
+      title: Other Features
+---
+
+{% include jumbotron.html body="**Direct**
+Direct is a payment service where Swedbank Pay helps improve cashflow by
+purchasing merchant invoices. Swedbank Pay receives invoice data, which is used
+to produce and distribute invoices to the consumer/end-use" %}
+
+## Introduction (NO, SE and FI)
+
+* To create an invoice payment, you need to collect all purchase information
+  and make a `POST` request towards Swedbank Pay.
+* You also need to collect social security number (SSN) / person number and
+  postal number from the consumer, and make another `POST` request towards
+  PayEx in order to retrieve the name and address from the consumer.
+* To create the authorization transaction, you need to calculate the final
+  price / amount, and make a third `POST` request where you send in
+  the consumer data.
+* To get the authorization result, you need  to follow up with a `GET`
+  request using the `paymentID` received in the first step.
+* Finally, when you are ready to ship your order, you will have to make
+  a `POST` request to make a Capture. **At this point Swedbank Pay will generate
+  the invoice to the consumer.**
```

Comment:
Creating a invoice statement in five easy steps:

1. Collect all purchase information send it to Swedbank Pay
2. Collect personal information and send it to PayEx
3. Create a authorization transaction
4. Retrieve the authorized payment
5. When the order is ready to be shipped, send a capture request